config/standalone-genesis.yaml

```diff
@@ -0,0 +1,9 @@
+account:
+  initBalances:
+    # overwrite with your test address
+    io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms: "100000000000000000000000000000000000"
```

Comment:
shall document the privkey of this addr? so easier for ppl to use

Reply:
We totally need the private key of the test account to be included, possibly in the same file, just as a comment above the address itself?

Reply:
this is just a placeholder, user should overwrite with their test address

Reply:
Yes makes sense. Yet I would still provide a very first test account with private key included so that I can include it in the docs and The user can get started with sending a transaction immediately, without having to browse fders and edit a config file. As long as it’s a Testnet, should be safe to have a test account with Private key included? Or it isn’t? Then for more complex scenarios the user can learn where to add custom initial accounts.